src/containers/edit-profile/profile-tab/profile-tab-form/ProfileTabForm.tsx

```diff
@@ -64,18 +64,15 @@ const ProfileTabForm: FC<ProfileTabFormProps> = ({
     handleNonInputValueChange('nativeLanguage', value)
   }
 
-  const resizeImage = (photo: File) => {
+  const resizeImage = async (photo: File) => {
     const originalPhotoPath = URL.createObjectURL(photo)
     const photoSizes = { newWidth: 440, newHeight: 440 }
 
-    imageResize(originalPhotoPath, photoSizes)
-      .then((resizedPhoto) => {
-        handleNonInputValueChange('photo', {
-          src: resizedPhoto,
-          name: photo.name
-        })
-      })
-      .catch(console.error)
+    const resizedPhoto = await imageResize(originalPhotoPath, photoSizes)
+    handleNonInputValueChange('photo', {
+      src: resizedPhoto,
+      name: photo.name
+    })
```

Comment:
What will happen if `imageResize` fails? Maybe use `try/catch` here?

Reply:
I've adder try/catch block an simulate error. Now looks like that
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/127191734/2b4ac9c7-3b8e-49dc-8b1e-5a61ad0211a2)